.github/workflows/ci-cd.yml

```
@@ -84,6 +84,12 @@ jobs:
     name: ⚙️ Pre-set global build settings
 
     runs-on: ubuntu-latest
+    # Prevent run 'push' events for the branches in upstream repository as it
+    # already covered by 'pull_request' event
+    if: >-
```

actionlint (1.7.4)

context "env" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)